src/shared/options/readOptions.ts

```diff
@@ -238,3 +243,37 @@ export async function readOptions(
 		options: augmentedOptions,
 	};
 }
+
+async function getBase(): Promise<OptionsBase> {
+	const commonScripts = ["lint:knip", "should-semantic-release", "test"];
+	const everythingScripts = [
+		"lint:md",
+		"lint:package-json",
+		"lint:packages",
+		"lint:spelling",
+	];
+
+	const scripts = await readPackageData().then((pkg) =>
+		pkg.scripts ? Object.keys(pkg.scripts) : [],
+	);
```

Comment:
[Style] Just nitpicking: what do you think of this smaller one-liner?

```suggestion
	const scripts = Object.keys((await readPackageData()).scripts ?? []);
```